nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/ListS3.java

```diff
@@ -217,6 +217,14 @@ public class ListS3 extends AbstractS3Processor implements VerifiableProcessor {
             .defaultValue("0 sec")
             .build();
 
+    public static final PropertyDescriptor MAX_AGE = new Builder()
```

Comment:
You need an additional validation in `customValidate` to ensure the MAX_AGE is greater than the MIN_AGE.

Reply:
thank you very much we added `createMaxAgeValidator` and fixed `testListIgnoreByMaxAge`, ready for review